core_components/bot_core/bot_core.go

```diff
@@ -27,7 +27,7 @@ var C = api.Component{
 	// Metadata
 	Code:         "bot_core",
 	Name:         "Bot Core",
-	Description:  "This component handles core routines and database management.",
+	Description:  "This component handles core routines and entity management.",
```

Comment:
Is there a difference between "_entity_" and "database_" management? Refactor issue?

Reply:
I'd stay with `entity` for this one. It sounds better, as the component only works with entites and entity managers. Database is abstracted away at this point.